tests/testthat/test-split_functional_groups.R

```diff
@@ -0,0 +1,81 @@
+# Instructions ----
+#' This file follows the format generated by ecosystemdata:::use_testthat_template().
+#' Necessary tests include input and output (IO) correctness [IO
+#' correctness], edge-case handling [Edge handling], and built-in errors and
+#' warnings [Error handling]. See `?ecosystemdata:::use_testthat_template` for more
+#' information. Every test should have a @description tag that takes up just
+#' one line, which will be used in the bookdown report of {testthat} results.
+
+# Setup ----
+# Load or prepare any necessary data for testing
+
+# split_functional_groups ----
+## IO correctness ----
+test_that("split_functional_groups() works with correct inputs", {
+  #' @description Test that split_functional_groups(x) returns a tibble.
+  expect_equal(
+    object = split_functional_groups(c("spiny dogfish0", "spiny dogfish 2")),
+    expected = tibble::tibble(
+      functional_group = c("spiny dogfish0", "spiny dogfish 2"),
+      species = c("spiny dogfish", "spiny dogfish"),
+      group = c("0", "2")
+    )
+  )
+
+  #' @description Test that split_functional_groups(x) returns y.
```

Comment:
Could you provide additional details in the description for this test?

Reply:
done